frontend/src/metabase/home/components/HomeContent/HomeContent.tsx

```diff
@@ -30,9 +26,6 @@ export const HomeContent = (): JSX.Element | null => {
   const { data: popularItems, error: popularItemsError } =
     usePopularItemListQuery({ reload: true });
   const error = databasesError || recentItemsError || popularItemsError;
-  const [showEmbedHomepage, setShowEmbedHomepage] = useState(() =>
```

Comment:
This is just removing the minimal version of the embedding homepage, which was a card below the xrays